src/aggregate/aggregate_request.c

```diff
@@ -168,6 +169,25 @@ int parseDialect(unsigned int *dialect, ArgsCursor *ac, QueryError *status) {
     return REDISMODULE_OK;
 }
 
+// Parse the available formats for search result values: FORMAT STRING|EXPAND
+int parseValueFormat(uint32_t *flags, ArgsCursor *ac, QueryError *status) {
+  const char *format;
+  int rv = AC_GetString(ac, &format, NULL, 0);
+  if (rv != AC_OK) {
+    QueryError_SetError(status, QUERY_EBADVAL, "Need an argument for FORMAT");
+    return REDISMODULE_ERR;
+  }
+  if (!strcasecmp(format, "EXPAND")) {
+    *flags |= QEXEC_FORMAT_EXPAND;
+  } else if (strcasecmp(format, "STRING")) {
+    QERR_MKBADARGS_FMT(status, "FORMAT %s is not supported", format);
+    return REDISMODULE_ERR;
+  }
+  *flags &= ~QEXEC_FORMAT_DEFAULT;
```

Comment:
Only if no format is specified, DEFAULT is set, and once the required value is calculated, it is equivalent to an explicit value and can continue as if it is not DEFAULT.
If explicitly requested EXPAND, only it will be set.
If explicitly requested STRING, none of them would be set.